var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -85,6 +94,9 @@ def cmake_args(self):
         if '~shared' in self.spec:
             options.append('-DBUILD_SHARED_LIBS:BOOL=OFF')
 
+        if '+static' in self.spec:
+            options.append('-DGMX_PREFER_STATIC_LIB:BOOL=ON')
+
```

Comment:
Please get rid of `shared` and `static`, and set up a multi-valued variant instead called `link_type`.   Follow example here: https://github.com/spack/spack/pull/11931